docs/docs/workflows/config.md

```diff
@@ -80,6 +80,50 @@ SNP-based attestation will be enabled as soon as a fix is verified.
 
 Fill the desired VM type into the **instanceType** fields in the `constellation-conf.yml` file.
 
+## Creating additional node groups
+
+By default, Constellation will create the node groups `control_plane_default` and `worker_default` for control-plane nodes and workers respectively.
```

Comment:
```suggestion
By default, Constellation creates the node groups `control_plane_default` and `worker_default` for control-plane nodes and workers, respectively.
```
Not from the style guide we follow, but IMO still good recommendation: "Use present tense for statements that describe general behavior that's not associated with a particular time." (https://developers.google.com/style/tense)